internal/integration/integration_test.go

```diff
@@ -92,24 +92,6 @@ func (suite *SpokesReceivePackTestSuite) TearDownTest() {
 	require.NoError(os.RemoveAll(suite.localRepo))
 }
 
-func (suite *SpokesReceivePackTestSuite) TestDefaultReceivePackSimplePush() {
-	assert.NoError(suite.T(), chdir(suite.T(), suite.localRepo), "unable to chdir into our local Git repo")
-	assert.NoError(
-		suite.T(),
-		exec.Command(
-			"git", "push", "--receive-pack=spokes-receive-pack", "r", "HEAD").Run(),
-		"unexpected error running the push with the default receive-pack implementation")
-}
-
-func (suite *SpokesReceivePackTestSuite) TestSpokesReceivePackSimplePush() {
```

Comment:
Is this test not still relevant?  I'm assuming the previous one was deleted because it tried to use spokes-receive-pack without the wrapper, which we expect not to work because it won't get the quarantine id that way.  But it seems like this one should probably still work.

Reply:
Thats a good catch. I assumed this test was an extension of the previous one that uses the default non-wrapper approach. I added the test back.